test/integration/puppetlabs/puppetserver/bootstrap_testutils.clj

```diff
@@ -39,6 +39,9 @@
 (def master-log-dir
   "./target/master-var/log")
 
+(def ca-dir
+  "./target/ca")
```

Comment:
In the fixtures directory we put the cadir under the confdir (though not in the ssldir).
eg `dev-resources/p/s/c_a/certificate_authority_int_test/ca_true_test/master/conf/ca/ca_pub.pem`

I would expect the equivalent for the fixtures within the ./target dir to be `./target/master-conf/ca/ca_pub.pem`.